lib/models/value_set_source.js

```diff
@@ -0,0 +1,72 @@
+const { ValueSetSchema } = require('cqm-models');
+const mongoose = require('mongoose');
+const _ = require('lodash');
+
+module.exports = class MongoDBValueSetSource {
+  /* Insert documentation here
+    */
+
+  constructor(connectionInfo) {
```

Comment:
The Measure Source constructor has `connectionInfo = null`, should this constructor act like the Measure source constructor?